lib/fixup.rb

```diff
@@ -1,5 +1,12 @@
+#!/usr/bin/env ruby
```

Comment:
Wrong file permission?

Reply:
Looks right?
```bash
chronos@cheekon-i686 /output/git/satmandu (more_glibc *$%)$ ls -aFl lib/fixup.rb
-rwxrwxr-x 1 chronos chronos 16284 Aug 20 19:53 lib/fixup.rb*
chronos@cheekon-i686 /output/git/satmandu (more_glibc *$%)$
```

Reply:
Does this need to be executable?

Reply:
I set it so that it could be run independently.  It can also be run with `ruby fixup.rb`.

I'm happy to remove the executable bit. Should I?

Reply:
done.

Reply:
Rubocop is complaining about permissions.  Unless we have a need to run as an executable, maybe it would be better to remove this line?